### PR TITLE
Fool Polymer 2.6+ into sending notifications

### PR DESCRIPTION
**Pull request by alshakero**
See https://github.com/Polymer/polymer/commit/e6d558ec

### CODE REVIEW

**Comment by alshakero** [comment]
Ready. Apps tests will not pass as they have bad `starcounter-include`.

File: juicy-html.html [hunk 1]
```diff
@@ -198,6 +198,10 @@
          * @type {Boolean}
          */
         JuicyHTMLElement.prototype.isAttached = false;
+        
+         // to fool Polymer into thinking this is is a Polymer element thus forwarding notifications to it.
+        JuicyHTMLElement.prototype.__dataHasAccessor = { model: true };
+        JuicyHTMLElement.prototype.__isPropertyEffectsClient = true;
```
Comment (tomalec) [inline juicy-html.html]: I'd add a test for this sneaky breaking change.

File: .travis.yml [hunk 2]
```diff
@@ -2,12 +2,12 @@ language: node_js
 sudo: required
 dist: trusty
 before_script:
-- npm install web-component-tester
-- npm install bower
-- npm install juicy-tattoo
+- yarn add web-component-tester
+- yarn add bower
+- yarn add juicy-tattoo
```
[the remark this answers is not in the document]
Comment (alshakero) [reply .travis.yml]: `npm` was down.

**Comment by alshakero** [comment]
Closed in favour of https://github.com/Juicy/imported-template/pull/56

**Pull request closed without merging**